src/Traits/FrozenTimeTrait.php

```diff
@@ -61,6 +61,10 @@ protected function stripTime($time)
      */
     public function setTime($hours, $minutes, $seconds = null, $microseconds = null)
     {
+        if (version_compare(PHP_VERSION, '7.1.0', '>=')) {
```

Comment:
can we make this a constant somewhere? Would like to avoid the comparison for each object created

Reply:
We could define a class/global constant inside of `src/carbon_compat.php`?

Reply:
Sounds good